serverlessworkflow/modules/ROOT/pages/eventing/handling-events-on-workflows.adoc

```diff
@@ -1,27 +1,22 @@
-= Handling events on workflows
+= Event state in Serverless Workflow
 
 :compat-mode!:
 // Metadata:
 :description: Handling events on workflows
 :keywords: kogito, workflow, serverless, event
 
-This document describes Serverless Workflow link:{spec_doc_url}#Event-State[Event state]. Event state can be used to start a new workflow instance upon the reception of a designated link:{cloud_events_url}[CloudEvent] or to pause the execution of an existing workflow instance till a designated CloudEvent is received. 
+This document describes link:{spec_doc_url}#Event-State[event state] and related examples in Serverless Workflow. You can use the event state to start a new workflow instance upon the reception of a designated link:{cloud_events_url}[CloudEvent] or to pause the execution of an existing workflow instance until a designated CloudEvent is received.
 
-[[eventMatch]]
-== Event State match 
+[[ref-example-event-match]]
+== Example of event match
 
-An Event state consist of one or more link:{spec_doc_url}#event-definition[event definitions]. Event definitions are combined to designate which CloudEvent Types the Event State should listen to. 
+An event state consists of one or more link:{spec_doc_url}#event-definition[event definitions]. Event definitions are combined to assign the CloudEvent types that the event state listens to.
```

Comment:
"assing" sound weird in this context, why "designate" is not right in your opinion? 
We are not assigning anything, we are indicating which events types the state should listen to.

Reply:
Done, changed the word to designate